modules/mgmt-power-remediation-baremetal-about.adoc

```diff
@@ -0,0 +1,114 @@
+
+[id="mgmt-power-remediation-baremetal-about_{context}"]
+= About power-based remediation of bare metal
+In a bare metal cluster, remediation of nodes is critical to ensure the overall health of the cluster. Physically remediating a cluster can be challenging and any delay in putting the machine to a safe or an operational state increases the time the cluster remains in a degraded state, and the risk that subsequent failures might bring the cluster offline. Power-based remediation helps counter such challenges.
+
+Instead of reprovisioning the nodes, power-based remediation uses a power controller to power off an inoperable node. This type of remediation is also called power fencing.
+
+The {product-title} uses the `MachineHealthCheck` controller to detect faulty bare metal nodes by initiating power-cycling. Power-based remediation is fast and reboots unhealthy hosts instead of removing them from the cluster.
+
+Power-based remediation meets the following objectives:
+
+* Allows the recovery of control plane nodes
+* Prevents data loss in hyperconverged environments.
+* Minimizes downtime for stateful applications that run on the unhealthy nodes.
+
+[id="machine-health-checks-bare-metal_{context}"]
+== MachineHealthChecks on bare metal
+
+Machine deletion on bare metal cluster triggers reprovisioning of a bare metal host.
+Usually bare metal reprovisioning is a lengthy process, during which the cluster
+is missing compute resources and applications might be interrupted.
+To change the default remediation process from machine deletion to host power-cycle,
+annotate the `MachineHealthCheck` resource with the
+`machine.openshift.io/remediation-strategy: external-baremetal` annotation.
+
+After you set the annotation, unhealthy machines are power-cycled by using
+BMC credentials.
+
+[id="mgmt-understanding-remediation-process_{context}"]
+== Understanding the remediation process
+
+The remediation process operates as follows:
+
+. The MachineHealthCheck (MHC) controller detects that a node is unhealthy.
+. The MHC notifies the controller in the cluster API provider for managed bare metal hardware (CAPM3) controller.
+. The controller in the cluster API provider for managed bare metal hardware (CAPM3) requests to power-off the unhealthy node.
+. After the power is off, the Node CR is deleted, which allows the cluster to reschedule the affected workload on other nodes.
+. The cluster API provider for managed bare metal hardware (CAPM3) requests to power on the node.
+. After the node is up, the node re-registers itself with the cluster, causing the creation of a new Node CR
+. After  the Node CR is recreated, the cluster API provider for managed bare metal hardware (CAPM3) restores the annotations and labels that existed on the unhealthy node before its deletion.
+
+[NOTE]
+====
+If the power operations did not complete, the cluster API provider for managed bare metal hardware (CAPM3) triggers the reprovisioning of the unhealthy node.
```

Comment:
unless this is a master node or node that can not be reprovisioned (e.g. `ExternallyProvisioned`)